framework/doc/content/source/bcs/DirichletBC.md

```diff
@@ -26,6 +26,12 @@ on the subset of the boundary denoted by $\partial \Omega_D$. In this case, the
 `value` corresponds to the constant $g$, and the user must define one
 or more sidesets corresponding to the boundary subset $\partial \Omega_D$.
 
+## Preset boundary conditions
```

Comment:
Might be good to merge the documentation into one place instead of having a separate ADFunctionDirichlet.md, DirichletBC.md, etc.